### ISSUE TITLE
[Programmatic Access – Make Code – Home Page] Narrator announces incorrect role information as 'Group' for 'View all' button under ‘Home Page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3555]
**User Experience:**
Users who depends upon screen reader get confused if the Screen reader reads incorrect role of any control. They'll not be able to know whether it is a group or a button or whether it is a non interactive control. Users will understand it to be a interactive control as screen reader announces group while navigating the control and hence it will not be a good usable experience for the users. 

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify that Narrator announces incorrect role information as 'Group' for 'View all' button or not.

**Actual Result:**
Narrator announces incorrect role information as 'Group' for 'View all' button under ‘Home Page’.

**Expected Result:**
Narrator should provide the correct role information for 'View all' control under ‘Home Page’ and it should be read as 'View all button'.

**Note:**
This issue does not exists on Firefox & Chrome.

**WCAG Reference:**
[WCAG 4.1.2 - Name, Role, Value
](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0)
Screen Reader: Narrator

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG4.1.2, #Win10-Edge(Chromium)

[WCAG4.1.2_Narrator announces incorrect role of view all button.zip](https://github.com/microsoft/pxt-microbit/files/5479507/WCAG4.1.2_Narrator.announces.incorrect.role.of.view.all.button.zip)
